flytekit/visualize/deck.py

```diff
@@ -0,0 +1,30 @@
+import json
+from typing import List, Union
+
+from flytekit import FlyteContextManager
+from flytekit.visualize.renderer import Renderer
+
+
+class Deck:
+    def __init__(self, name: str, renderers: Union[List[Renderer], Renderer]):
+        self.name = name
+        self.renderers = renderers if isinstance(renderers, list) else [renderers]
+        ctx = FlyteContextManager.current_context()
+        ctx.user_space_params.decks.append(self)
+
+    def append(self, r: Renderer) -> "Deck":
+        self.renderers.append(r)
+        return self
+
+    def upload(self, path):
```

Comment:
what is upload? it should not have this method in the user facing api. It should be in dispatch_execute right?